src/client/dfuse/pil4dfs/int_dfs.c

```diff
@@ -599,19 +598,19 @@ discover_daos_mount(void)
 
 	pool = getenv("DAOS_POOL");
 	if (pool == NULL) {
-		D_WARN("warning> DAOS_POOL is not set.\n");
+		D_WARN("DAOS_POOL is not set.");
 		return;
 	}
 
 	container = getenv("DAOS_CONTAINER");
 	if (container == NULL) {
-		D_WARN("warning> DAOS_CONTAINER is not set.\n");
+		D_WARN("DAOS_CONTAINER is not set.");
 		return;
 	}
 
 	pt_end = stpncpy(dfs_list[num_dfs].fs_root, fs_root, DFS_MAX_PATH - 1);
 	if ((long int)(pt_end - dfs_list[num_dfs].fs_root) >= (DFS_MAX_PATH - 1)) {
-		D_WARN("dfs_list[num_dfs].fs_root[] is not large enough.\nQuit\n");
+		D_WARN("dfs_list[num_dfs].fs_root[] is not large enough.");
```

Comment:
Needs a newline.

This reads like a debug message, what does it mean when this triggers and what should the user do about it?  It would be good to reword the message here.

Reply:
Fixed as suggested. Thank you!